packages/transition-backend/src/services/transitRouting/TrRoutingBatch.ts

```diff
@@ -258,38 +258,19 @@ class TrRoutingBatch {
         routing: TransitRouting
     ): Promise<{ input: string; csv?: string; detailedCsv?: string; geojson?: string }> => {
         const { resultHandler, pathCollection } = await this.prepareResultData(routing);
-
-        let currentResultPage = 0;
-        let totalCount = 0;
-        const pageSize = 250;
-        do {
-            const { totalCount: total, tripResults } = await resultsDbQueries.collection(this.options.jobId, {
-                pageIndex: currentResultPage,
-                pageSize
+        const resultStream = resultsDbQueries.streamResults(this.options.jobId);
+        for await (const row of resultStream) {
```

Comment:
I absolutely need them all... This one is because the stream is an async iterator. I'm pretty sure it's this part that was causing the OOM (the 250 elements page for a batch routing calculation with alternatives), so this PR streams the database results instead of paging them.

Reply:
If you'd change the resultParser to operate on a stream and return a stream, you could just pipe everything, but yeah, that's a much bigger/complex change. 
(Could keep a //TODO around)

Reply:
I added a TODO that we can look at when we do result refactoring. For now though, I just hope to see Hubert's job complete with success with this PR. If it doesn't work, we can revert.